isdown/README.md

```diff
@@ -0,0 +1,55 @@
+# Agent Check: IsDown
+
+## Overview
+
+[IsDown][1] is a status page aggregator & outage monitoring tool that helps businesses stay on top of their critical dependencies. Provide your team real-time monitoring and instant notifications for outages in all your tools and cloud providers. We monitor more than 2000 status pages.
+
+What IsDown offers:
+
+* Get outage alerts in Datadog from all your third-party dependencies ( SaaS, PaaS, Cloud Providers, etc.)
+* Understand the frequency of outages through your different providers
+* Have the external outages together with the internal information
```

Comment:
Would it be possible to clarify or add details about the `internal information` here?

Reply:
I've added more information.